draft-ietf-tls-dtls13.md

```diff
@@ -272,34 +281,62 @@ verify the TLS MAC.  The DTLS record format is shown below:
 
    struct {
        ContentType opaque_type = 23; /* application_data */
-       ProtocolVersion legacy_record_version = {254,253); // DTLSv1.2
-       uint16 epoch;                         // DTLS-related field
-       uint48 sequence_number;               // DTLS-related field
+       uint32 epoch_and_sequence;
        uint16 length;
        opaque encrypted_record[length];
-} DTLSCiphertext;
+    } DTLSCiphertext;
 ~~~~
 
 type:
 : The content type of the record.
 
-legacy_record_version:
-: This field is redundant and it is treated in the same way as specified in the TLS 1.3 specification.
-  The DTLS version 1.2 version number is reused, namely { 254, 253 }.
-  This field is deprecated and MUST be ignored.
-
-epoch:
-: A counter value that is incremented on every cipher state change.
-
-sequence_number:
-: The sequence number for this record.
+epoch_and_sequence:
+: The low order two bits of the epoch and the low order 30 bits of
+  the sequence number, laid out as a 32 bit integer with the
+  epoch constituting the first two bits and the sequence number
```

Comment:
"The first 2 bits hold low order bits from the epoch and the remaining 30 bits hold the low order bits from the sequence number."  Or something like that.